perffile/events.go

```diff
@@ -75,30 +79,37 @@ func (e eventUnknown) Generic() EventGeneric {
 }
 
 // EventHardware represents a hardware event.
+type EventHardware struct {
+	ID        EventHardwareID
+	PMUTypeID uint32
```

Comment:
I wonder if PMUTypeID should have its own type, even if it's just an opaque uint32 right now. I'm a little surprised there aren't defined constants for that already, and they might show up in the future.

Reply:
Oh, that is confusing. Do you happen to know if there's some new metadata block in the perf.data that gives you the meanings of these IDs, then?

I think I would still lean toward having a type, even if it's not an enumeration, because these are still nominal numbers.

Reply:
I figured it out after enough digging, thanks for pushing. See https://github.com/aclements/go-perf/pull/4/commits/f34eb848c4e0355382859b60a79a65a9d44ac8e4 for details.